templates/ingress/ingress.yaml

```diff
@@ -64,10 +64,12 @@ spec:
   - http:
       paths:
 {{- if semverCompare "<1.19-0" .Capabilities.KubeVersion.GitVersion }}
+      {{- if ne .Values.expose.ingress.controller "alb" }}
```

Comment:
What's the difference between the `if ne .Values.expose.ingress.controller "alb"` and `eq .Values.expose.ingress.controller "alb"` code block?

Reply:
@ywk253100 The important thing is where portal_path `/*` is set in the ingress configuration for alb-load-balancer-controller. if it is the first thing, then all requests go to portal_path even though other path patterns such as `api/*` exists. To avoid this, when it is an alb load balancer, I set it as the last rule.

I do not know how the other ingress work and didn't want to break them. Therefore, when ingress type is not `alb`, portal_path is the 1st rule (like before) and if it is `alb`, it is the last rule

Reply:
Per my understanding, as the portal path `/*` can match all requests, if it works as the first rule, it should work as the last one.
So how about putting the portal path as the last one for all kinds of ingress controller?

Reply:
@ywk253100 sure. I will update the PR